daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -39,9 +46,10 @@ func New(ctx logger.Context) (logger.Logger, error) {
 		return nil, err
 	}
 	return &JSONFileLogger{
-		f:   log,
-		buf: bytes.NewBuffer(nil),
-		ctx: ctx,
+		f:       log,
+		buf:     bytes.NewBuffer(nil),
+		ctx:     ctx,
+		readers: pubsub.NewPublisher(3*time.Second, 0),
```

Comment:
Okay, I'm not sure. But timer in `pubsub` very slow and heavy, it can be okay I think for now.